aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/internal/ServiceUtils.java

```diff
@@ -302,7 +302,7 @@ public static void downloadToFile(S3Object s3Object,
             }
             outputStream = new BufferedOutputStream(new FileOutputStream(
                     dstfile, appendData));
-            byte[] buffer = new byte[1024*10];
```

Comment:
`8192` isn't _that_ difficult to figure out but it might be worth putting the magic number into a well named constant variable.